browser/multiFa.go

```diff
@@ -22,7 +22,7 @@ func MultiFaVisualizer(infile string, outfile string, start int, end int, noMask
 	if noMask {
 		fasta.AllToUpper(records)
 	}
-	for i := range records {
+	for i := 1; i < len(records); i++ {
```

Comment:
here a range statement was added incorrectly by me, as this loop should be from 1 to n, not 0 to n. Fixed here.